src/styles/cards.scss

```diff
@@ -0,0 +1,118 @@
+@import "./variables.scss";
+
+body,
+h2,
+p,
+.page {
+  margin: 0;
+}
+
+.main {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  width: 4 * $card-width + 3 * $card-margin;
+  padding-top: 50px;
+  margin: auto;
+}
+
+.card {
+  font-family: Roboto, sans-serif;
+  width: #{$card-width};
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  margin-right: #{$card-margin};
+  margin-bottom: 46px;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
fix it everywhere